test/e2e/vault-decryption-chrome.spec.ts

```diff
@@ -155,6 +155,32 @@ async function closePopoverIfPresent(driver: Driver) {
   await driver.clickElementSafe(nftAutodetection);
 }
 
+async function switchToMetaMaskWindowWithoutSocket(driver) {
```

Comment:
I would strongly prefer this logic to be written in
`test/e2e/webdriver/driver.js`
and/or
`test/e2e/background-socket/window-handles.ts`
rather than in the individual spec file.

Reply:
sure make sense! I placed it here as it's the only test that uses a prod build, but agree that more tests can come in the future and could also need to utilize this function. I'll change this now, thank you!! :pray: